src/register/Register.jsx

```diff
@@ -35,12 +40,11 @@ export default function Register() {
         let data = "";
         fetch(`${BASE_API}/register`, requestRegister)
             .then(async response => {
-
                 data = await response.json();
-                setResponseMessage(data["message"]);
+                setResponseMessage(data.message);
             })
             .catch(error => {
-                setResponseMessage(data["message"]);
+                setResponseMessage(data.message);
```

Comment:
What is the expected output here? if the call fails, and we end up in the catch block, what would data.message be?

Reply:
I've modified the fetch block
<img width="477" alt="Screen Shot 2020-07-31 at 8 59 21 pm" src="https://user-images.githubusercontent.com/29667122/89029009-c8fc5780-d370-11ea-9bd1-d9ae5994dc26.png">

Thee responseMessage set inside if block (line 45) is if the response is 201, for all other customized responses (400, 409. 500) will be set as response on line 46. The line 49 for inside catch is reserve for any unexpected responses such as 500 that is returned when BIT backend server is not running.

Reply:
@ramitsawhney27 We need to refactor all api calls in one place. But that will be a separate PR.